assets/prototype/domain/shared/entities/datetimes/predicates/sorters/index.test.ts

```diff
@@ -0,0 +1,58 @@
+import { head, last } from 'ramda';
+import { compareAsc, parseISO } from 'date-fns';
+
+import sorters from './index';
+import { DatesSortedBy } from '../../../../../eventEditor/data/date/types';
+import { nodes as dates } from '../../../../../eventEditor/data/queries/datetimes/test/data';
+
+describe('sorters', () => {
+	it('should return dates sorted in default chronological order if no sort prop is provided', () => {
+		const sortedDates = sorters({ dates });
+		const firstDate = head(sortedDates);
+		const lastDate = last(sortedDates);
```

Comment:
what about looping through the array and calling `compareAsc()` on the current and previous elements of the array?
you would simply have to start the loop on the second element. 
That way we could confirm the entire array was sorted correctly.

Reply:
Done: https://github.com/eventespresso/event-espresso-core/pull/2277/commits/d88817f07238f8cba90da56e3e17909fe94a4720
Ended up being a bit imperative, but didn't had a cleaner solution, but since these are tests, it's good enough IMO.